src/UKHO.S100PermitService.Common/Events/EventIds.cs

```diff
@@ -138,6 +138,21 @@ public enum EventIds
         /// <summary>
         /// 840027 - Exception occurred while get permit key from Product Key Service
         /// </summary>
-        ProductKeyServicePostPermitKeyRequestFailed = 840027
+        ProductKeyServicePostPermitKeyRequestFailed = 840027,
+
+        /// <summary>
+        /// 840028 - Manufacturer Id not found for Manufacturer keys in Memory cache.
```

Comment:
Update description as         /// 840028 - Manufacturer Id not found for Manufacturer keys in Key Vault.

Reply:
done